test_porcupine.py

```diff
@@ -0,0 +1,51 @@
+from porcupine import *
+import sys
+import pytest
+sys.path.append('./')
```

Comment:
Is it necessary to add the current directory to the path? It should be there already